jwst/pipeline/calwebb_spec2.py

```diff
@@ -660,6 +665,70 @@ def _process_grism(self, data):
         calibrated = self.photom.run(calibrated)
         return calibrated
 
+    def _process_miri_wfss(self, data):
+        """
+        Calibrate MIRI WFSS  data.
+
+        Determine the order of the steps
+
+        Parameters
+        ----------
+        data : JWSTDataModel
+            The input science data model.
+
+        Returns
+        -------
+        JWSTDataModel
+            The calibrated data model.
+        """
+        # Apply flat-field correction - we do not do this for MIRI WFSS.
+        # After more data is taken we might come back to this. Leaving the
+        # code commented out as a reminder.
+        # calibrated = self.flat_field.run(data)
```

Comment:
Have you considered just keeping this in the pipeline but then setting it to skip by default using the parameter reference file?  The benefit would be that it'd be easier to test the effects of the step without requiring additional code changes later.

Reply:
Added flat_field back in for testing. If there is no flat found in CRDS the step is skipped.